### DISCUSSION TITLE
Is it possible to show DSO labes without tips?

### DISCUSSION THREAD

**Discussion by AardeGeoWorld** [Stellarium/stellarium#3443] [Q&A]
I'm interested in showing DSO names - without the tips (small red ellipse around M31; small green rectangle & yellow dots around Pleiades; etc.) Would that be possible?
![Screenshot 2023-10-03 232638](https://github.com/Stellarium/stellarium/assets/84190981/e9488e69-708c-487b-a851-6e77d3114d8c)

**Comment by alex-w** [comment]
No

**Reply by AardeGeoWorld** [reply]
OK, that's a clear reply :)
